src/modules/map/components/openlayers/OpenLayersMap.vue

```diff
@@ -139,6 +139,7 @@ export default {
             /** Keeping trace of the starting center in order to place the cross hair */
             initialCenter: null,
             popoverCoordinates: [],
+            animationDuration: IS_TESTING_WITH_CYPRESS ? 0 : this.animationDuration,
```

Comment:
shouldn't we define a solid value here? I don't see where `this.animationDuration` could give us the `250` we had before

Reply:
Oh sorry yes, that's what I wanted, I fixed it now.